### ISSUE TITLE
Redo the Burp config

### ISSUE THREAD

**Issue by parsiya** [parsiya/Parsia-Clone#11]
Seems like some parts of the Burp config has changed, redo them.

E.g., this is to disable intercept at startup.

```json
    "user_options":{
        "misc":{
            "enable_proxy_interception_at_startup":"never"
        }
    },
```

`user_options` might have a new place. Create a new config that has everything we want including but not limited to the TLS passthrough domains for proxying Windows using system settings.

**Comment by parsiya** [comment]
Reduced the size in 433583e17a0ea9cc096c305d43c9223892da94b1.

Not sure, if that is the correct approach because it means I cannot modify the config manually and add new items. Tbh, I doubt anyone cares about the size of the config at 20KB vs 8KB.

**Comment by parsiya** [comment]
This was not a great idea, the default filters went away.
Try removing t he shortcut keys to make it smaller, maybe the shortcut keys are hardcoded?

If not, just do it like before. Who cares about the size

**Comment by parsiya** [comment]
Fixed in 8a42b7a7cf9e0ab5be5b454ba9a5ca0161f478b5. Also added user options to disable intercept at startup
